tests/v1/tpu/test_basic.py

```diff
@@ -63,3 +63,40 @@ def test_basic(
         output = vllm_outputs[0][1]
 
         assert "1024" in output or "0, 1" in output
+
+
+@pytest.mark.skipif(not current_platform.is_tpu(),
+                    reason="This is a test for TPU only")
```

Comment:
This test needs to also check that there are 8 devices available for tp=8

Reply:
Makes sense. Fixed. Thanks for the suggestion.